alf/algorithms/rl_algorithm.py

```diff
@@ -605,11 +634,18 @@ def _process_unroll_step(self, policy_step, action, time_step,
         exp = make_experience(time_step.cpu(),
                               alf.layers.to_float32(policy_step),
                               alf.layers.to_float32(policy_state))
-
+        effective_unroll_steps = 1
         store_exp_time = 0
         if not self.on_policy:
+            rollout_info = policy_step.info
+            # 1) process
+            post_processed_exp_list = self.post_process_experience(
+                rollout_info, time_step.step_type, exp)
+            effective_unroll_steps = len(post_processed_exp_list)
```

Comment:
In order really get this right, need to do: `sum(exp.step_type.shape[0] for exp in post_processed_exp_list) / exp.step_type.shape[0]`

Reply:
Updated.